CatAnalyzer/scripts/catGetDatasetInfo

```diff
@@ -44,6 +45,7 @@ for l in csv[1:]:
                'xsec':float(xsec),
                'nevt':int(nevt),
                'lumi':float(lumi),
+               'colour':int(colour),               
```

Comment:
In the MakeHist.py, this can be improved with eval() statement. By this, 'kRed+1' can be addressed.